uxarray/__init__.py

```diff
@@ -1,2 +1,17 @@
 from .grid import *
 from .helpers import *
+
+
+def __version__():
+    # Attempt to import the needed modules
+    try:
+        from importlib.metadata import version as version
+    except ImportError:
+        from importlib_metadata import version as version
+
+    try:
+        __version = version("uxarray")
+        print(__version)
+    except AttributeError:
+        # Incase something happens, such as the library isn't installed
+        __version = "0000"
```

Comment:
Actually, I'd suggest a placeholder like 999, as some code that uses ours as a dependency might have minimum version checks, which we don't need to trigger if the only thing wrong here is pulling our version, does that make sense?

Reply:
Yeah, that makes perfect sense, I hadn't thought of that at all, thanks!